terraform/main.mk

```diff
@@ -44,7 +57,6 @@ terraform.debug:
 terraform.init: gomplate terraform
 	@ \
  	cd $(ENV_DIR) && \
- 	mkdir -p $(ENV_DIR)/.terraform && \
```

Comment:
@gloovers please check with @kobrikx because this line was added to avoid issues OR using current user approach fixed that issue?

Reply:
we should remove this row since running terraform container with non-root user create `.terraform` folder with correct permission
@kobrikx ?

Reply:
Seems like this should be safe to remove